kubernetes/cmsweb/scripts/test_k8s_aps.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+##H Usage: test_k8s_aps.sh <base_url>
+##H
+
+if [ "$1" == "-h" ] || [ "$1" == "-help" ] || [ "$1" == "--help" ] || [ "$1" == "help" ] || [ "$1" == "" ]; then
+    perl -ne '/^##H/ && do { s/^##H ?//; print }' < $0
```

Comment:
Please do not make dependency on perl, it requires to have perl installed on a system just for one line parsing. Instead use standard UNIX tools like grep, sed, awk. For example, this line can be replaced with `grep "##H" $0 | grep -v grep | sed -e "s,##H,,g"` and it does not require perl at all.